deploy/protocol.ts

```diff
@@ -27,6 +27,11 @@ const deployProtocol: DeployFunction = async (hre) => {
   const loansEscrowBeacon = await deployLoansEscrowBeacon(hre)
   const collateralEscrowBeacon = await deployCollateralEscrowBeacon(hre)
   const tTokenBeacon = await deployTTokenBeacon(hre)
+  const nftDictionary = await contracts.get('TellerNFTDictionary')
+
+  const tokens = getTokens(network)
+  const wrappedNativeToken =
+    networkName == 'polygon_mumbai' ? tokens.erc20.WMATIC : tokens.erc20.WETH
```

Comment:
will want this to also include `polygon` mainnet network as well

Reply:
changed to mainnet since Mumbai isn't feasible